pkg/etcdenvvar/etcd_env.go

```diff
@@ -26,10 +26,10 @@ type envVarContext struct {
 }
 
 var FixedEtcdEnvVars = map[string]string{
-	"ETCD_DATA_DIR":              "/var/lib/etcd",
-	"ETCD_QUOTA_BACKEND_BYTES":   "7516192768", // 7 gig
-	"ETCD_INITIAL_CLUSTER_STATE": "existing",
-	"ETCD_ENABLE_PPROF":          "true",
+	"ETCD_DATA_DIR":               "/var/lib/etcd",
+	"ETCD_QUOTA_BACKEND_BYTES":    "7516192768", // 7 gig
+	"ETCD_INITIAL_CLUSTER_STATE":  "existing",
+	"ETCD_ENABLE_PPROF":           "true",
```

Comment:
to enable debug logs on serverside set 

`ETCD_DEBUG` to true

Reply:
Thanks, will try that one now.